rtsf-at-checkout-rsp-controller-event-handler/eventhandler/process_events.go

```
@@ -18,61 +17,52 @@ const (
 	RFIDEventMoved    = "moved"
 	ROIActionEntered  = "ENTERED"
 	ROIActionExited   = "EXITED"
-)
 
-var loggingClient logger.LoggingClient
+	resourceName = ""
+	sourceName   = "rfid-roi-event"
+	deviceName   = "device-rfid-roi-rest"
+	profileName  = ""
+)
 
 // ProcessRspControllerEvents transforms RSP Controller events to an RFID ROI (region or interest), as defined by checkout-event-reconciler
-func ProcessRspControllerEvents(edgexcontext *appcontext.Context, params ...interface{}) (bool, interface{}) {
-	if len(params) < 1 {
-		// We didn't receive a result
-		return false, nil
-	}
-
-	loggingClient = edgexcontext.LoggingClient
+func ProcessRspControllerEvents(edgexcontext interfaces.AppFunctionContext, data interface{}) (bool, interface{}) {
+	lc := edgexcontext.LoggingClient()
 
-	result, ok := params[0].(models.Event)
+	event, ok := data.(dtos.Event)
 	if !ok {
-		loggingClient.Error("No event received by RSP Controller event handler")
+		lc.Error("No event received by RSP Controller event handler")
```

Return the error rather than log it.